packages/flutter/lib/src/rendering/platform_view.dart

```diff
@@ -383,6 +383,9 @@ class RenderUiKitView extends RenderBox {
     if (event is! PointerDownEvent) {
       return;
     }
+    if (!hasSize) {
+      return;
+    }
```

Comment:
This fix is not complete, if the view is laid out and got move offstage, it will still have size and intercept tap

for example: https://gist.github.com/chunhtai/388f1160663796297f05c182a49fe512
If you tap the ion on the top right and toggle the webview to the front and then toggle it to the back. If you tap the screen, the RenderUiKit will still intercept the pointer. It looks like we may need a bigger refactor...

Reply:
Ah, now I understand better what you were saying in #83481. :thinking: Thank you.